backend/lightning/lightning.go

```diff
@@ -62,6 +65,10 @@ func (lightning *Lightning) Activate() error {
 		return errp.New("Lightning node already active")
 	}
 
+	if len(lightningConfig.Accounts) > 0 {
+		return errp.New("Lightning accounts already configured")
+	}
+
```

Comment:
i just saw now that the check above kind of does the same, but then i wonder if `lightnignConfig.Inactive` still makes sense after this commit? (see other comment)